spark/src/main/java/org/apache/iceberg/actions/RewriteManifestsAction.java

```diff
@@ -153,227 +74,13 @@ public RewriteManifestsAction stagingLocation(String newStagingLocation) {
    * @return this for method chaining
    */
   public RewriteManifestsAction useCaching(boolean newUseCaching) {
-    this.useCaching = newUseCaching;
+    delegate.option("use-caching", Boolean.toString(newUseCaching));
```

Comment:
Make a constant owned by RewriteManifestsAction?

Reply:
I should have added such a constant to `BaseRewriteManifestsSparkAction` for sure like I did in the action for expiring snapshots. Let me fix that. I am not sure about this place, though.

Reply:
Added to `BaseRewriteManifestsSparkAction` for now.